jsr352/core/src/test/java/org/hibernate/search/jsr352/massindexing/RestartChunkIT.java

```diff
@@ -57,6 +56,8 @@
 })
 public class RestartChunkIT {
 
+	static final int BM_COUNT = 100;
```

Comment:
Maybe it would be better if the name was a bit more explicit? Something like "ITEMS_BEFORE_SIMULATED_FAILURE"?